README.md

```diff
@@ -270,6 +270,47 @@ awxJob:
     }
 ```
 
+### Alertmanager Configuration
+
+Follow the upstream [Prometheus Alertmanager documentation](https://prometheus.io/docs/alerting/configuration/)
+to configure alerts.
+
+For reference, here is an example Alertmanager configuration that sends
+an alert to Autoheal with authentication. This example assumes autoheal
```

Comment:
... to the auto-heal service with ...

That is how we call ourselves: the auto-heal service.

Reply:
Done.